src/wazuh_modules/wmodules.c

```diff
@@ -111,21 +111,29 @@ int wm_check() {
         for (j = prev = wmodules; j != i; j = next) {
             next = j->next;
 
-            if (i->context->name == j->context->name) {
-                mdebug1("Deleting repeated module '%s'.", j->context->name);
+            if(i->tag && j->tag){
+                if(!strcmp(i->tag,"command") || !strcmp(j->tag,"command")){
```

Comment:
This should not be mandatory at this point. We should warn about it in the configuration reader.